### PR TITLE
Add condition id in step for conditional workflows

### PR DESCRIPTION
**Pull request by nipierre**
- Modification of the schema to allow keyword `condition` in the step
- Allow conditional workflows

### CODE REVIEW

File: standard/1.3/workflow-definition.schema.json [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+                "name": {
+                    "examples": [
+                        "job_transfer"
+                    ],
+                    "pattern": "^([a-z_]+)$",
+                    "title": "Name of the parameter",
+                    "type": "string"
+                },
+                "condition": {
+                    "title": "Boolean condition, if true, starts the step, if false, skips.",
```
Comment (MarcAntoine-Arnaud) [inline standard/1.3/workflow-definition.schema.json:143]: description will be more:
```
Expression which requires to return a boolean. On false condition, the step will be skipped.
```

File: standard/1.3/workflow-definition.schema.json [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        "version_major",
+        "version_minor",
+        "version_micro",
+        "tags",
+        "parameters",
+        "steps"
+    ],
+    "additionalProperties": false,
+    "title": "Schema of Workflow Definition for Media Cloud AI",
+    "type": "object"
```
Comment (MarcAntoine-Arnaud) [inline standard/1.3/workflow-definition.schema.json:261]: change reference on documentation:
https://github.com/media-cloud-ai/media-cloud-ai.github.com/blob/master/parameter.schema.md
and here:
https://github.com/media-cloud-ai/media-cloud-ai.github.com/blob/master/workflow-definition.schema.md

File: standard/1.3/workflow-definition.schema.json (hunk 1 above)
Comment (nipierre) [inline standard/1.3/workflow-definition.schema.json:143]:
```suggestion
                    "title": "Expression which requires to return a boolean. On false condition, the step will be skipped.",
```

File: standard/1.3/workflow-definition.schema.json (hunk 2 above)
Comment (nipierre) [reply standard/1.3/workflow-definition.schema.json:261]: Updated

**Pull request merged**